pkg/util/workerqueue/workerqueue_test.go

```diff
@@ -194,3 +195,14 @@ func TestWorkerQueueEnqueueAfter(t *testing.T) {
 		assert.Fail(t, "should have got a queue'd message by now")
 	}
 }
+
+func TestIsDebugError(t *testing.T) {
```

Comment:
a nit but this doesn't just test IsDebugError any longer, so it should be renamed to TestDebugError instead.